src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/InterchangeHeader.java

```diff
@@ -30,6 +30,8 @@
     private static final int TRANSLATION_TIME_INDEX = 4;
     private static final int SEQUENCE_NUMBER_INDEX = 5;
     private static final int NHSACK_INDEX = 9;
+    private static final int SENDER_ID_INDEX = 0;
+    private static final int RECIPIENT_ID_INDEX = 0;
```

Comment:
Would it be possible to rename the ID_INDEX constants to something that distinguishes them from the others? Since the ID_INDEXs are working on a subsection of the segment whereas the others are working on the whole segment?

Reply:
Sure, i've renamed these to `SENDER_SUBSECTION_ID_INDEX` and `RECIPIENT_SUBSECTION_ID_INDEX`. Hope that makes more sense, otherwise let me know if you have any suggestions

Reply:
Works for me!